webapp/management/commands/sendmail.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management.base import BaseCommand, CommandError
+
+import json
+from webapp.mailer.mailer import Mailer
+
+
+class Command(BaseCommand):
+    help = 'Send email'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'test',
+            help="Send test email",
+            nargs='+',
+            type=str
+            )
+
+    def handle(self, *args, **options):
+        self.stdout.write('Sending email')
+
+        mail_data = json.loads(
+            open('webapp/mailer/liam.tset', 'r+').read()[::-1]
```

Comment:
Not quite sure what kind of file `liam.tset` is, but I'm assuming it has your mailgun account/key stored and can therefore be extracted by anyone by opening the file in python. Let's update this a bit in class to have the credentials in `config/settings/base.py` so we can just reference them by calling `settings.MAILGUN_ACCOUNT`